source-browser.html

```diff
@@ -166,28 +191,57 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
               ),
             );
         });
-
+          
         $('#topics, #loading').toggleClass('display-none');
       }
-
+        
       function refreshTopics() {
         localStorage.clear();
         showTopics();
       }
-
+        
       function showSearchInput() {
         $('#search')
           .removeClass('display-none')
           .focus();
       }
-
+        
+        
+        var array = ['languages', 'trunk', 'staging', 'nursery', 'incubator', 'tools'];
+        array.forEach(function(header) {
+          $("#navbar").append("<li><a href=#"+header+" id=li"+header+">"+header+"</a></li>");
```

Comment:
There should be no concatenation here.

Reply:
Try using the href and text methods. jQuery elements can also take a list of attributes. Nested elements should be constructed via append for readability.